src/app/asset-page/asset-page.component.ts

```diff
@@ -924,10 +924,10 @@ export class AssetPage implements OnInit, OnDestroy {
       referring_requestid: this._search.latestSearchRequestId,
       ab_segments: abSegments ? [abSegments] : [],
       item_id: assetId,
+      ...doi && {doi: [doi]},
```

Comment:
What is the purpose of the `...`? Are we trying to do a spread?

Reply:
It's actually being used to conditionally add the `doi` key to the top level dictionary. Learned here: https://medium.com/@mikeh91/conditionally-adding-keys-to-javascript-objects-using-spread-operators-and-short-circuit-evaluation-acf157488ede

Let me know if that is overly confusing though. The alternative would probably be logging an empty array or `null`?

Reply:
Ahh gotcha. Makes sense now. 👍